### PR TITLE
[Doc] Add missing `Admin`'s notification prop

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: docs/Admin.md [hunk 1]
```diff
@@ -533,6 +533,33 @@ const App = () => (
 );
 ```
 
+## `notification`
```
Comment (fzaninotto) [inline docs/Admin.md:536]: please add it to the list of props at the beginning of this chapter, too

File: docs/Admin.md [hunk 2]
```diff
@@ -533,6 +533,33 @@ const App = () => (
...
+
+You can override the notification component, for instance to change the notification duration. It defaults to 4000, i.e. 4 seconds, and you can override it using the `autoHideDuration` prop. For instance, to create a custom Notification component with a 5 seconds default:
```
Comment (fzaninotto) [inline docs/Admin.md:538]:
```suggestion
You can override the notification component, for instance to change the notification duration. A common use case is to change the `autoHideDuration`, and force the notification to remain on screen longer than the default 4 seconds. For instance, to create a custom Notification component with a 5 seconds default:
```

File: docs/Admin.md [hunk 3]
```diff
@@ -533,6 +533,33 @@ const App = () => (
...
+
+```jsx
+// in src/MyNotification.js
+import { Notification } from 'react-admin';
+
+const MyNotification = props => <Notification {...props} autoHideDuration={5000} />;
```
Comment (fzaninotto) [inline docs/Admin.md:544]:
```suggestion
const MyNotification =() => <Notification autoHideDuration={5000} />;
```

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/Admin.md (hunk 2 above)
Comment (WiXSL) [reply docs/Admin.md]: This was taken from the recipes section, should I change it there too?

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**